src/Doctrine/Repository/PaymentRepositoryInterface.php

```diff
@@ -0,0 +1,12 @@
+<?php
+
+declare(strict_types=1);
+
+namespace AdeoWeb\SyliusEveryPayPlugin\Doctrine\Repository;
+
+use Sylius\Component\Payment\Model\PaymentInterface;
+
+interface PaymentRepositoryInterface
+{
+    public function getByEveryPayReference(string $reference): ?PaymentInterface;
```

Comment:
```suggestion
    public function findOneByEveryPayReference(string $reference): ?PaymentInterface;
```
tik ką refactorinau visas repozitorijas ir kad būtų consistent visur dabar tokius metodus pervadinau iš get į find ir jeigu tik vieną returnina, tai findOne